cmd/utils/dev.go

```diff
@@ -212,98 +202,10 @@ func GetDevFromManifest(manifest *model.Manifest, devName string) (*model.Dev, e
 		return nil, err
 	}
 
-	return manifest.Dev[devKey], nil
-}
-
-//AskYesNo prompts for yes/no confirmation
-func AskYesNo(q string) (bool, error) {
```

Comment:
this has been moved to prompt/questions.go as is logic to get input from the user